src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -289,26 +313,37 @@ public class ReleaseService : IReleaseService
                     var oldStatus = release.ApprovalStatus;
 
                     release.ApprovalStatus = request.ApprovalStatus;
-                    release.InternalReleaseNote = request.InternalReleaseNote;
+                    release.InternalReleaseNote = request.LatestInternalReleaseNote;
                     release.NextReleaseDate = request.NextReleaseDate;
 
                     release.PublishScheduled = request.PublishMethod == PublishMethod.Immediate &&
                                                request.ApprovalStatus == ReleaseApprovalStatus.Approved
                         ? DateTime.UtcNow
                         : request.PublishScheduledDate;
 
+                    var releaseStatus = new ReleaseStatus
```

Comment:
I think you should make updates to the ReleaseService tests for this as tests already exist for UpdateRelease.